### PR TITLE
Promote Pod Disruption Budgets to GA

### PR DESCRIPTION
**Pull request by mortent**
PR to update documentation after PDBs have been promoted to GA.

KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/85-Graduate-PDB-to-Stable/README.md
Issue: https://github.com/kubernetes/enhancements/issues/85
PR: https://github.com/kubernetes/kubernetes/pull/99290

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/disruptions.md
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/disruptions.md:93]: The shortcode works like this:
```suggestion
{{< feature-state for_k8s_version="v1.21" state="stable" >}}
```

File: content/en/docs/tasks/run-application/configure-pdb.md
Comment (sftim) [inline content/en/docs/tasks/run-application/configure-pdb.md:9]:
```suggestion
{{< feature-state for_k8s_version="v1.21" state="stable" >}}
```

File: content/en/docs/tasks/run-application/configure-pdb.md
Comment (sftim) [inline content/en/docs/tasks/run-application/configure-pdb.md:117]: style nit:
```suggestion
The behavior for an empty selector differs between the policy/v1beta1 and policy/v1 APIs for
PodDisruptionBudgets. For policy/v1beta1 an empty selector matches zero pods, while
for policy/v1 an empty selector matches every pod in the namespace.
```

**Review by sftim — DISMISSED** [review]
Hi @mortent. Would you be willing to just change the English localization (`en`) here? SIG Docs localization teams pick up these changes independently from one another and each team can pick its own different process for managing their backlog.
We'll need this PR to omit changes to other localizations before we can merge it.
/hold

You might also want to check about including the changes for issue https://github.com/kubernetes/website/issues/26385 in this update. It's also OK if those come in as 2 PRs.


Optional extra tidying
- add `min-kubernetes-server-version: 1.21` to the front matter of `content/en/docs/tasks/run-application/configure-pdb.md`  
  and add `{{< version-check >}}` into the prerequisites section
    - this is because older clusters don't have a PDB API available as `policy/v1`
-  remove “(available in Kubernetes 1.7 and higher)” as v1.7 is a long time out of support

**Comment by mortent** [comment]
@sftim I have removed the changes to any other languages than English and updated the PR based on your suggestions.

**Comment by reylejano** [comment]
Hi @mortent , thank you for having your Doc PR ready for review, here are upcoming doc related dates for the 1.21 release:
Upcoming doc related dates for the 1.21 release:
- <strike> Docs Ready for Review deadline is March 24</strike>
- Docs Ready to Merge deadline is March 31

**Comment by reylejano** [comment]
@kubernetes/sig-apps-pr-reviews  , please provide a technical review for this PR by March 31 to get this into the release. Thank you!

File: content/en/docs/tasks/run-application/configure-pdb.md
Comment (sftim) [inline content/en/docs/tasks/run-application/configure-pdb.md:123]: Consider omitting the “For versions 1.8 and earlier” note as v1.8 is long out of support.

Comment (mortent) [reply content/en/docs/tasks/run-application/configure-pdb.md]: Removed the section.

File: content/en/docs/concepts/workloads/pods/disruptions.md
Comment (annajung) [inline content/en/docs/concepts/workloads/pods/disruptions.md:93]: Can you make sure to make changes to the feature gate page, remove `PodDisruptionBudget` from [Feature gates for Alpha or Beta features](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features) to [Feature gates for graduated or deprecated features](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-graduated-or-deprecated-features). When removing from the alpha/beta table, please make sure to remove include the alpha/beta history in the GA table. Thanks!

File: content/en/docs/tasks/run-application/configure-pdb.md
Comment (sftim) [inline content/en/docs/tasks/run-application/configure-pdb.md:21]: I recommend (not essential, but a good idea) that we ask people using older Kubernetes versions to check the documentation for that Kubernetes version.

File: content/en/docs/concepts/workloads/pods/disruptions.md
Comment (mortent) [reply content/en/docs/concepts/workloads/pods/disruptions.md:93]: Updated

File: content/en/docs/tasks/run-application/configure-pdb.md
Comment (mortent) [reply content/en/docs/tasks/run-application/configure-pdb.md:21]: Not sure I understand what we are looking for here. Is there an example I can look at?

**Pull request merged**
